### DISCUSSION TITLE
v.exe blocked by windows during http request

### DISCUSSION THREAD

**Discussion by definitelynico** [vlang/v#13772] [General]
I'm new to V and pretty new to programming in general. I was playing around today, did the "v up" and created a new project. I decided to try making a http request, pretty simple, grab the date/time from "https://vlang.io/utc_now", format it and print to console. When I ran the program Windows decided to block "v.exe", declaring it "Trojan:Win32/Bearfoos.B!ml". I'm guessing that this has nothing to do with V itself but Windows being Windows.

Do I need to pay any attention to this or can I safely just restore the executable?

Update: I reinstalled V, tried to recreate what happened with no success. I might have missed the "make.bat" last time, honestly can't remember. I'll provide a screenshot anyway.

![vlang windows defender trojan detectino](https://user-images.githubusercontent.com/101659586/159076181-f686805d-f2ad-4db5-888a-282b2a70291f.png)

**Comment by ylluminate** [comment]
EDIT: I see your update now, thanks for clarifying. 

--------------- 

Sounds like a false positive. Also, probably should report such issues within https://github.com/vlang/v/issues with details instead of using Discussions here as I think it's a better format for such a report.

For example, if I were reporting this I'd capture all of the OS specific info (`v doctor`) and provide the sample code that reproduces this "detection" when you execute V (run or compile? Etc.)  Armed with this info a V developer that uses Windows with a matching config or has a virtual machine with such a config could try to reproduce and maybe even resolve this.

Also feel free to chat about such concerns in Discord: https://discord.gg/vlang

**Reply by definitelynico** [reply]
Thank you! I'll look into doing that. I was just a little scared opening an issue knowing that this is probably not a problem with V itself.

**Comment by Wajinn** [comment]
For those using the Windows OS, it is arguably also good to report false-positives to Microsoft ([www.microsoft.com/en-us/wdsi/filesubmission](http://www.microsoft.com/en-us/wdsi/filesubmission)). This will also protect against those attempting to make false reports for dishonest reasons. There is also a category for software developers, so perhaps the contributors or developers of V might want to use that. You will have to sign in with a Microsoft or Skype account, or make a new one.
